src/multirust/telemetry.rs

```diff
@@ -0,0 +1,34 @@
+use time;
+use config::Cfg;
+use multirust_utils::utils;
+use rustc_serialize::json;
+
+#[derive(Debug, PartialEq)]
+pub enum TelemetryMode {
+    On,
+    Off,
+}
+
+#[derive(RustcDecodable, RustcEncodable, Debug)]
+pub enum TelemetryEvent {
+    RustcRun { duration_ms: u64, exit_code: i32, errors: Option<String> },
+    ToolchainUpdate { toolchain: String, success: bool } ,
+    TargetAdd { toolchain: String, target: String, success: bool },
+}
+
+#[derive(RustcDecodable, RustcEncodable, Debug)]
+struct LogMessage {
+    log_time_s: i64,
+    event: TelemetryEvent,
+}
+
+pub fn log_telemetry(event: TelemetryEvent, cfg: &Cfg) {
+    let ln = LogMessage { log_time_s: time::get_time().sec, event: event };
```

Comment:
:open_mouth: With as much as I've been using `now_utc` and various `Timespec` functions in example code I've written, I should've caught this. Will fix shortly.